### ISSUE TITLE
New Item:  Light Bridge Redirection Cube

### ISSUE THREAD

**Issue by WhoozYerBuddy** [BEEmod/BEE2-items#2367]
A cube that will not set on top of a light bridge but instead will drop through the bridge and redirect it in the same way that a laser redirection cube redirects a laser.  Along with the light bridge receiver/relay this could make for some really interesting light bridge bootstrapping puzzles.

**Comment by IceTudor** [comment]
Nice idea! What about a funnel redirection cube? That could work, after all, the laser, light bridge and funnel emit in the same way I think.

**Comment by WhoozYerBuddy** [comment]
I thought about a funnel redirection cube also.  It could be effective.  The behavior is a little different with the objects in a funnel moving.  IMO, objects shouldn't go through a funnel redirection cube, but it would still give some fun options.

**Comment by IceTudor** [comment]
Yeah. I also thought of a new laser/hard light/funnel etc item. An electric arc which would either be in a deactivated, off or on state, like antlines, and would send a short pulse of electricity when powered.

**Comment by TeamSpen210** [comment]
The problem with this is that we can't really change the fact that bridges are solid, so it'll end up passing through the cube.

**Comment by WhoozYerBuddy** [comment]
I don't know how any of it works on the coding side of things, but how do lifts stop light bridges when extended?  Could that effect be used on a cube (that would be movable)?  Basically the cube would be a movable combination of light bridge receiver and light bridge emitter.  Both already exist in static form.  Is the moveable quality the sticking point?

**Comment by ellingtonvengaboysmuseum** [comment]
it's impossible to redirect light bridges without some seriously complicated coding that probably valve would have to do themselves...

**Comment by WhoozYerBuddy** [comment]
Is it the movable quality that is the sticking point?

**Comment by ellingtonvengaboysmuseum** [comment]
the only way to do this would be to make a custom cube that would be huge and attach a light bridge emmiter to it and have the bridge turned off when the cube is moved to prevent massive lag...

**Comment by WhoozYerBuddy** [comment]
"Attach a light bridge emitter to it" - Are you talking about the emitter model?  Why would that need to be attached.  There are certainly different emitter models, I'm sure you could have the bridge without the emitter or rather with an invisible emitter.  Why would there be massive lag with the light bridge if it moved?  Is this merely conjecture on your part or have you actually done any testing?

**Comment by TeamSpen210** [comment]
Light Bridges and Funnels were not designed to be moved. So they cannot be. You can move the emitter (the model), but the bridge itself doesn't move. To fix that you need to turn it off and on again. The problem is that cleans gel off the bridge. The lag would be from rapidly turning it off and on, while it's moving.

**Comment by WhoozYerBuddy** [comment]
I see.  Thanks for the explanation.

**Comment by IceTudor** [comment]
Is it possible to attach world portals onto the cube so that the light bridge goes to the correct place? Probably not, just asking.

**Comment by Endermage77** [comment]
I don't think so, because the cube is supposed to catch a light bridge from any angle.
